src/Stormpath/Cache/Cacheable.php

```diff
@@ -0,0 +1,55 @@
+<?php namespace Stormpath\Cache;
+
+abstract class Cacheable {
+
+    protected function addDataToCache($data, $href)
+    {
+        $key = $this->generateKey($href);
+        $this->cache->put($key, $data, $this->cacheManager->options['ttl']);
+    }
+
+    protected function cachedData($href)
+    {
+        $key = $this->generateKey($href);
+        return $this->cache->get($key);
+    }
+
+    protected function createCachableResource($resource, $parentHref)
```

Comment:
What is this method doing? Looks like it assumes it has a tenant, which is not correct. It also deletes an entry in a method that, if I understand correctly, should be creating something.